cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,17 @@
+/// <reference types="cypress" />
+
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
 
+    cy.visit('user/register');
+    cy.get(':nth-child(1) > .form-control').type(user.username);
```

Comment:
It's recommended to use more specific selectors rather than `:nth-child` which can easily break if the structure of the form changes. Consider assigning `data-*` attributes to your form fields and use them as selectors. This will make your tests more resilient and maintainable.

Reply:
How i can use data-*